.github/workflows/terragrunt-apply-production.yml

```diff
@@ -54,29 +55,12 @@ jobs:
         working-directory: env/cloud/ecr
         run: terragrunt apply --terragrunt-non-interactive -auto-approve
 
-  detect-lambda-changes:
-    needs: terragrunt-apply-ecr-only
-    runs-on: ubuntu-latest
-    outputs:
-      lambda-to-rebuild: ${{ steps.filter.outputs.changes }}
-    steps:
-      - name: Checkout
-        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
-
-      - name: Filter
-        id: filter
-        uses: dorny/paths-filter@de90cc6fb38fc0963ad72b210f1f284cd68cea36 # v3.0.2
-        with:
-          filters: .github/lambda-filter.yml
-
   build-tag-push-lambda-images:
-    needs: detect-lambda-changes
```

Comment:
Just to keep this from running at the same time as the ECR apply, we should set this:
```
needs: terragrunt-apply-ecr-only
```

Reply:
Good catch!